packages/contracts/package.json

```diff
@@ -12,12 +12,11 @@
     "dist"
   ],
   "scripts": {
-    "typechain": "find ./artifacts -name '*.json' -type f -not -path './artifacts/@aragon/*' -not -path './artifacts/build-info/*' | grep -v '.dbg.json' | xargs typechain --target=ethers-v5 --out-dir ./typechain",
+    "typechain": "find ./artifacts -name '*.json' -type f -not -path './artifacts/@aragon/osx-v1.0.1/*' -not -path './artifacts/build-info/*' | grep -v '.dbg.json' | xargs typechain --target=ethers-v5 --out-dir ./typechain",
```

Comment:
Do we need to change the package in two places if we want to test against a different aragon/osx version?

Another thing to double-check: v1.0.1 was meant to be 1.0.0, correct?

Reply:
`Do we need to change the path/alias in two places if we want to test against a different aragon/osx version?` no we can keep appending new version. 

`Another thing to double-check: v1.0.1 was meant to be 1.0.0, correct?`
yes, v1.0.0 do not exist, we start from v1.0.1

Reply:
but i think it would be better to have one source for the aliases
check [945b6bc](https://github.com/aragon/osx/pull/418/commits/945b6bc8ac6e2ac18ad9bd9d2be19b4252bacd8d)

Reply:
Nice!